presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/security/FileBasedAccessControl.java

```diff
@@ -318,6 +319,12 @@ public void checkCanShowRoleGrants(ConnectorSecurityContext context, String cata
     {
     }
 
+    @Override
+    public void checkCanExecuteProcedure(ConnectorSecurityContext context, SchemaProcedureName procedure)
+    {
+        //TODO implement
```

Comment:
Remove this. I'm not sure it makes sense to implement this generically.

Reply:
i asked for this & asked for a follow-up too